config/bidderinfo.go

```diff
@@ -22,6 +22,11 @@ type BidderInfo struct {
 	GVLVendorID             uint16               `yaml:"gvlVendorID"`
 	Syncer                  *Syncer              `yaml:"userSync"`
 	Experiment              BidderInfoExperiment `yaml:"experiment"`
+	CompressionType         *CompressionType     `yaml:"compressionType"`
+}
+
+type CompressionType struct {
+	EndpointCompression string `yaml:"endpointCompression"`
```

Comment:
Can we add a small description for `CompressionType`? Please feel free to reword as you see fit.
```
27 + // CompressionType if set, determines the type of compression the bid request will undergo before being sent to the corresponding bid server
28   type CompressionType struct {
29       EndpointCompression string `yaml:"endpointCompression"`
30   }
31  
```